crates/cxx-qt-lib/src/types/qdate.rs

```diff
@@ -37,11 +40,15 @@ mod ffi {
         #[doc(hidden)]
         #[rust_name = "qdate_init"]
         fn construct(y: i32, m: i32, d: i32) -> QDate;
+
+        #[doc(hidden)]
+        #[rust_name = "qdate_to_qstring"]
+        fn qdateToQString(date: &QDate) -> QString;
```

Comment:
Probably don't need a cpp bridge for this, a straight CXX binding will likely work as there is no difference between Qt 5 / 6? So something like `#[rust_name = "to_string"] fn toString(self: &QDate) -> QString` ?

Reply:
I tried that, but the Qt `QDate::toString()` doesn't have a variant with no argumens. There are variants with default arguments, but I couldn't get cxx to understand that.

The following variants are available:
- `QString toString(const QString &format, QCalendar cal = QCalendar()) const`
- `QString toString(Qt::DateFormat format = Qt::TextDate) const`
- `QString toString(QStringView format, QCalendar cal = QCalendar()) const`

I suspect we would want to use the second one,but again cxx complained that this function didn't exist if I called it with no arguments. I also tried calling it with an integer (since Qt::DateFormat is an enum) but that didn't work either, any ideas how to get it working?

Reply:
Ah right it sometimes gets confused. But see my other comment I wonder whether a generic template for all Qt types could work as well, maybe something to discuss when I'm back in Jan :-)